cypress/support/step-definitions/menu-steps.js

```diff
@@ -135,3 +136,19 @@ Then("Menu {string} expandable element has inner elements", (position) => {
     .should("have.attr", "data-component", "link")
     .and("be.visible");
 });
+
+// instructions for search in menu keyboard nav test
+When("I press {string} key {int} times", (key, times) => {
+  cy.focused().trigger("keydown", keyCode(key));
+  for (let i = 0; i < times; i++) {
+    cy.focused();
+  }
+});
+
+Then("Search component input should be focused", () => {
+  cy.get(SEARCH_COMPONENT).find("input").should("have.focus");
```

Comment:
for example `searchDefault()` from `cypress/locators/search/index.js`

Reply:
I think `searchInput` will be a better option from the Search locators file. In this use case as it already has `cy.get(SEARCH_COMPONENT).find("input")` and I would just need to append the `.should("have.focus");`